demo/app/Prescriptor.py

```diff
@@ -3,10 +3,42 @@
 import pandas as pd
 import numpy as np
 from keras.models import load_model
-from unileaf_util.framework.transformers.data_encoder import DataEncoder
+from sklearn.preprocessing import MinMaxScaler
 
 from . import constants
 
+class Encoder:
+    """
+    A mini version of the unileaf-util DataEncoder.
```

Comment:
No need to mention "unileaf" here. You're the only one that's used this DataEncoder.